--- vendor/github.com/gardener/gardener/hack/install-requirements.sh	2021-02-10 20:16:45.159871502 +0100
+++ hack/install-requirements.sh	2021-02-11 18:35:18.483190897 +0100
@@ -22,7 +22,7 @@
 
 export GO111MODULE=on
 curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin v1.27.0
-curl -s "https://raw.githubusercontent.com/helm/helm/v2.16.9/scripts/get" | bash -s -- --version 'v2.13.1'
+curl -s "https://raw.githubusercontent.com/helm/helm/v2.17.0/scripts/get" | bash -s -- --version 'v2.17.0'
 
 if [[ "$(uname -s)" == *"Darwin"* ]]; then
   cat <<EOM
